lib/hammer_cli_katello/host_extensions.rb

```diff
@@ -57,18 +58,28 @@ module HostExtensions
                   _("Content View Environment Labels"), Fields::Field
             collection :content_view_environments, _('Content View Environments') do
               from :content_view do
-                label _("Content view") do
-                  field :id, _("Id")
-                  field :name, _("Name")
-                  field :composite, _("Composite"), Fields::Boolean
-                  field :rolling, _("Rolling"), Fields::Boolean
+                # Deprecated label. To be removed in future versions.
+                label _("Content view"), :sets => ['ALL'] do
+                  field :id, _("Id"), Fields::Field, :replaced_by_path => ["!p", "!p", "id"]
```

Comment:
Looking at your follow up PR (https://github.com/theforeman/hammer-cli/pull/397) I found it over-complicated for just translation purposes. I decided to write an alternative, but noticed that it also gets uglier. What if we rather re-create the path ourselves? Due to the nature of deprecation (it should be rather temporal anyway), we could WET ourselves a bit. What do you think about:
```suggestion
                  field :id, _("Id"), Fields::Field, :replaced_by => [_('Content Information'), _('Content View Environments'), _('CV Id')].join('/')
```

It could be improved by extracting the strings (or rather `_()` return value) into a variable and re-used within output definition as well as for `replaced_by` option. The `join`ing could be aslo extracted into a helper method, something like `def deprecated_labels_path(*args); args.join('/'); end`.

Reply:
Thanks for taking another look at this. I'm all right with this suggested change; it looks like the translations get automatically pulled in when the string matches one to one:
```
Warning: Field 'Informations sur le contenu/Content View Environments/Affichage de contenu/Id' is deprecated. Consider using 'Informations sur le contenu/Content View Environments/CV Id' instead.
```

Seems like the risk of incorrect translations is essentially zero here. I'll submit PRs for the needed changes and let you know when things are good to go :+1: